### PR TITLE
[merged] generate an UUID value for system containers

### PR DESCRIPTION
**Pull request by giuseppe**

### CODE REVIEW

File: Atomic/syscontainers.py [hunk 1]
```diff
@@ -419,6 +420,8 @@ def _do_checkout(self, repo, name, img, upgrade, values, destination, unitfileou
                 key, val = i[:split], i[split+1:]
                 values[key] = val
 
+        if "UUID" not in values:
+            values["UUID"] = str(uuid.uuid4())
```
Comment (rhatdan) [inline Atomic/syscontainers.py]: Does this generate the UUID IN a form that systemd likes?  Or in the Docker format?

Comment (giuseppe) [inline Atomic/syscontainers.py]: it generates an UUID in the NNNNNNNN-NNNN-NNNN-NNNN-NNNNNNNNNNNN format that is the format expected by systemd for $container_uuid

File: Atomic/syscontainers.py [hunk 2]
```diff
@@ -591,7 +594,7 @@ def update(self, name):
         values = info["values"]
         revision = info["revision"] if "revision" in info else None
 
-        if revision and self.args.setvalues is None:
+        if revision and self.args.setvalues is None and not self.args.force:
```
Comment (giuseppe) [inline Atomic/syscontainers.py]: I added a fixup patch, that move the not force condition.  And took the occasion to add a comment, for what we are doing here

File: Atomic/syscontainers.py (hunk 1 above)
Comment (rhatdan) [inline Atomic/syscontainers.py]: great

**Comment by rhatdan** [comment]
@rh-atomic-bot r+

**Pull request closed without merging**